pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -122,7 +121,7 @@ func (configgen *ConfigGeneratorImpl) buildOutboundClusters(env *model.Environme
 			} else {
 				// set TLSSettings if configmap global settings specifies MUTUAL_TLS, and we skip external destination.
```

Comment:
Just for tracking, can you add TODO to remove this else branch as part of deprecation of env.Mesh.AuthPolicy flag. I think it's ok to remove now, though may be safer to do in the next point release.

(tracking issue https://github.com/istio/istio/issues/5900)